applications/nrf5340_audio/README.rst

```diff
@@ -0,0 +1,922 @@
+      To use the proprietary LC3 audio codec, you need to obtain a license.
+      For more information, see `requirements <octave_requirements>`_.
+
+#. The audio decoder decodes the data and sends the uncompressed audio data (PCM) back to the :file:`audio_datapath.c` module.
+#. The :file:`audio_datapath.c` module continuously feeds the uncompressed audio data to the hardware codec.
+#. The hardware codec receives the uncompressed audio data over the inter-IC sound (I2S) interface and performs the digital-to-analog (DAC) conversion to an analog audio signal.
+
+.. _octave_requirements:
+
+Requirements
+************
+
+The nRF5340 Audio application is designed to be used only with the following custom hardware:
+
++---------------------+----------+--------------------------+---------------------------------+
+| Hardware platforms  | PCA      | Board name               | Build target                    |
++=====================+==========+==========================+=================================+
+| nRF5340 Audio       | PCA10121 | nrf5340_audio_dk_nrf5340 | nrf5340_audio_dk_nrf5340_cpuapp |
++---------------------+----------+--------------------------+---------------------------------+
+
+You need at least two nRF5340 Audio development kits (one with the gateway firmware and one with headset firmware) to test the application.
+For CIS with TWS in mind, three kits are required.
+
+Software codec requirements
+===========================
+
+The nRF5340 Audio application must use either the LE Audio's LC3 software or the open source SBC software codec known from Classic Bluetooth Audio.
+Each codec requires :ref:`adding its own repository before building and running <octave_configuration_repos>`.
+
+|lc3_codec_access_note|
+See :ref:`octave_configuration_select_codec` for more information.
+
+.. _octave_dk:
+
+nRF5340 Audio DK
+================
+
+The nRF5340 Audio development kit is a hardware development platform that demonstrates the nRF5340 Audio application.
+
+.. _octave_dk_drawings:
+
+Hardware drawings
+-----------------
+
+The nRF5340 Audio hardware drawings show both sides of the development kit, as delivered with the case:
+
+.. figure:: /images/nRF5340_audio_dk_front_case.svg
+   :alt: Figure 1. nRF5340 Audio DK (PCA10121) front view
+
+   Figure 1. nRF5340 Audio DK (PCA10121) front view
+
+.. figure:: /images/nRF5340_audio_dk_back_case.svg
+   :alt: Figure 2. nRF5340 Audio DK (PCA10121) back view
+
+   Figure 2. nRF5340 Audio DK (PCA10121) back view
+
+The following figure shows the back of the development kit without the case:
+
+.. figure:: /images/nRF5340_audio_dk_back.svg
+   :alt: Figure 3. nRF5340 Audio DK (PCA10121) back view without case
+
+   Figure 3. nRF5340 Audio DK (PCA10121) back view without case
+
+.. _octave_requirements_build_types:
+
+nRF5340 Audio build types
+=========================
+
+The nRF5340 Audio uses combinations of multiple :file:`.conf` files for different build types.
```

Comment:
Using `build type` term here may be a bit confusing. See: https://developer.nordicsemi.com/nRF_Connect_SDK/doc/latest/nrf/ug_multi_image.html for details. Maybe we can name it in an other way.